services/replay/service.go

```diff
@@ -780,9 +799,22 @@ func (s *Service) handleReplay(w http.ResponseWriter, req *http.Request) {
 	}
 	replay, err := s.replays.Get(id)
 	if err != nil {
-		httpd.HttpError(w, "could not find replay: "+err.Error(), true, http.StatusNotFound)
+		httpd.HttpError(w, "could not find replay: "+id, true, http.StatusNotFound)
```

Comment:
Why the change to hide the error?

Reply:
The reasoning is that 'could not find replay:  + id' would be enough to understand the problem. But I can add it back  since it may help to debug the cause of the problem.